drivers/src/persistent.rs

```diff
@@ -237,6 +238,8 @@ pub struct PersistentData {
     #[cfg(feature = "runtime")]
     pub attestation_disabled: U8Bool,
     #[cfg(feature = "runtime")]
+    pub runtime_cmd_active: U8Bool,
+    #[cfg(feature = "runtime")]
```

Comment:
This isn't necessarily the most correct place to store this bool. At the same time, I don't want to allocate a big new chunk for this.

Thoughts?

Reply:
Since it's related to disabled attestation, I think this is fine.